pkg/morph/event/notary_preparator.go

```diff
@@ -43,6 +44,9 @@ var (
 
 	// ErrMainTXExpired is returned if received fallback TX is already valid.
 	ErrMainTXExpired = errors.New("received main tx has expired")
+
+	// ErrUnknownEvent is returned if main transaction is not expected.
+	ErrUnknownEvent = errors.New("received notary request is not allowed")
```

Comment:
Don't we need to add a special case for this event to `func (l *listener) parseAndHandleNotary(nr *result.NotaryRequestEvent) {`? We may probably safely skip logging for these events as they are unexpected and unwanted (currently `could not prepare and validate notary event` warning will be logged).

Reply:
@carpawell, could you push the update?

Reply:
sure, that was an error, i thout about extending `switch` but forgot about it